engine/frontend/src/App.js

```diff
@@ -1,13 +1,15 @@
 import './App.css';
 import Home from './component/Home';
+import { ChakraProvider } from '@chakra-ui/react'
 
 const App = () => {
-    console.log("Enclave Manager version: 2023-08-28-2")
```

Comment:
@Peeeekay leave this in here so we can track the versions to make sure we know what version people are running in the wild.